cmd/kube-apiserver/app/server.go

```diff
@@ -272,6 +272,7 @@ func Run(s *options.APIServer) error {
 	genericConfig.ProxyDialer = proxyDialerFn
 	genericConfig.ProxyTLSClientConfig = proxyTLSClientConfig
 	genericConfig.Serializer = api.Codecs
+	genericConfig.OpenAPIInfo.Title = "Kubernetes"
```

Comment:
newConfig set both Title and Version to "Generic API Server" and "unversioned". here we just overwrite the title.